afl-plot

```diff
@@ -137,94 +124,105 @@ set key outside
+set xlabel \"relative time in seconds\" font \"small\"
+"
+
+PLOT_HF="
+set terminal png truecolor enhanced size 1000,300 butt
+set output '$outputdir/high_freq.png'
+
+$GNUPLOT_SETUP
 
 plot '$inputdir/plot_data' using 1:4 with filledcurve x1 title 'total paths' linecolor rgb '#000000' fillstyle transparent solid 0.2 noborder, \\
      '' using 1:3 with filledcurve x1 title 'current path' linecolor rgb '#f0f0f0' fillstyle transparent solid 0.5 noborder, \\
      '' using 1:5 with lines title 'pending paths' linecolor rgb '#0090ff' linewidth 3, \\
      '' using 1:6 with lines title 'pending favs' linecolor rgb '#c00080' linewidth 3, \\
      '' using 1:2 with lines title 'cycles done' linecolor rgb '#c000f0' linewidth 3
+"
 
-set term x11 window "$W_ID3"
-set output
-replot
+PLOT_LF="
+set terminal png truecolor enhanced size 1000,200 butt
+set output '$outputdir/low_freq.png'
 
-pause mouse close
+$GNUPLOT_SETUP
 
-_EOF_
+plot '$inputdir/plot_data' using 1:8 with filledcurve x1 title '' linecolor rgb '#c00080' fillstyle transparent solid 0.2 noborder, \\
+     '' using 1:8 with lines title ' uniq crashes' linecolor rgb '#c00080' linewidth 3, \\
+     '' using 1:9 with lines title 'uniq hangs' linecolor rgb '#c000f0' linewidth 3, \\
+     '' using 1:10 with lines title 'levels' linecolor rgb '#0090ff' linewidth 3
+"
 
-) | gnuplot &
+PLOT_ES="
+set terminal png truecolor enhanced size 1000,200 butt
+set output '$outputdir/exec_speed.png'
 
-(
+$GNUPLOT_SETUP
 
-cat << _EOF_
-#set xdata time
-#set timefmt '%s'
-#set format x "%b %d\n%H:%M"
-set tics font 'small'
-unset mxtics
-unset mytics
+plot '$inputdir/plot_data' using 1:11 with filledcurve x1 title '' linecolor rgb '#0090ff' fillstyle transparent solid 0.2 noborder, \\
+     '$inputdir/plot_data' using 1:11 with lines title '    execs/sec' linecolor rgb '#0090ff' linewidth 3 smooth bezier;
+"
 
-set grid xtics linetype 0 linecolor rgb '#e0e0e0'
-set grid ytics linetype 0 linecolor rgb '#e0e0e0'
-set border linecolor rgb '#50c0f0'
-set tics textcolor rgb '#000000'
-set key outside
+PLOT_EG="
+set terminal png truecolor enhanced size 1000,300 butt
+set output '$outputdir/edges.png'
 
-set autoscale xfixmin
-set autoscale xfixmax
+$GNUPLOT_SETUP
 
-set xlabel "relative time in seconds" font "small"
+plot '$inputdir/plot_data' using 1:13 with lines title '        edges' linecolor rgb '#0090ff' linewidth 3
+"
 
-set terminal png truecolor enhanced size 1000,200 butt
-set output '$outputdir/low_freq.png'
 
-plot '$inputdir/plot_data' using 1:8 with filledcurve x1 title '' linecolor rgb '#c00080' fillstyle transparent solid 0.2 noborder, \\
-     '' using 1:8 with lines title ' uniq crashes' linecolor rgb '#c00080' linewidth 3, \\
-     '' using 1:9 with lines title 'uniq hangs' linecolor rgb '#c000f0' linewidth 3, \\
-     '' using 1:10 with lines title 'levels' linecolor rgb '#0090ff' linewidth 3
+if [ "$#" = "3" ] && ( [ "$3" = "-g" ] || [ "$3" = "--graphical" ] ); then
```

Comment:
IMHO it is fine for the `else` of this print to give an informational message that "-g" would give a graphical UI if afl-plot-ui is built and installed. otherwise people wont know about it.